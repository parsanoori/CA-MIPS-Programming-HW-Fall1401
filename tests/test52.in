22
261036624
840402099
451075174
513811479
994331029
548857210
131264674
1039497362
96750264
638359767
458568953
930858466
946971062
945096883
448605615
768304219
591597079
53836642
1057526333
1005897537
466132212
27034435