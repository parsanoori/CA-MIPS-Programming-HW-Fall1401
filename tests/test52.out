27034435
53836642
96750264
131264674
261036624
448605615
451075174
458568953
466132212
513811479
548857210
591597079
638359767
768304219
840402099
930858466
945096883
946971062
994331029
1005897537
1039497362
1057526333